scripts/enable-ecs-agent-gpu-support.sh

```diff
@@ -22,9 +23,74 @@ enabled=1
 exclude=libglvnd-*
 EOF
 
-# this repo is temporary and only used for installing the system-release-nvidia package
```

Comment:
why are we getting rid of this? system-release-nvidia installs an nvidia repo into `/etc/yum.repos.d/` which is what we were supposed to be using to install the nvidia packages, this change would opt us out of installing from the system-release-nvidia repo which I don't think we want to do

Reply:
see line 94 below -- we're still using the system-release-nvidia from the amzn2-nvidia repo.

I'll add these comments back down there. 

Essentially we're adding amzn2-nvidia then pulling the nvidia version number only
then we're disabling amzn2-nvidia in favor of the cuda-rhel7 repo to pull the open driver and build it.
then we're disabling cuda-rhel7 and re-enabling amzn2-nvidia (again) to build everything as usual.

Reply:
I see, so the point being that the installed "open driver" will be the exact same version number that they would otherwise get from the AL repos?